app/components/SheetMusic/comments/Comments.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import fluxMixin from 'flummox/mixin';
+import Comment from './Comment';
+import ReplyBox from './ReplyBox';
+
+function retrieveInitialData(flux, id) {
+  let sheetMusicActions = flux.getActions('sheetmusic');
+  return Promise.all([
+    sheetMusicActions.getComments(parseInt(id, 10), flux),
+  ]);
+}
+
+export default React.createClass({
+  propTypes: {
+    id: React.PropTypes.String,
+  },
+
+  mixins: [
+    fluxMixin({
+      comments: store => ({
+        comments: store.state.comments,
+      }),
+    }),
+  ],
+
+  componentDidMount() {
+    retrieveInitialData(this.flux, this.props.id);
+  },
+
+  render() {
+    let comments = this.state.comments.comment ? this.state.comments.comment : [];
+    return (
+      <div className="comment__style">
+        <If condition={comments.length}>
+          {comments.slice(0, comments.length).map((comment) => (
```

Comment:
Not sure if the `slice` operation is necessary, you're always going to get the entire array when you do `comments.slice(0, comments.length)`.